Xlib/ext/nvcontrol.py

```diff
@@ -0,0 +1,5358 @@
+# the memory clock is stored in the lower 16 bits of the integer.
+# All clock values are in MHz.  All clock values are in MHz.
+#
+
+NV_CTRL_GPU_CURRENT_CLOCK_FREQS = 93  # R--G
+
+#
+# NV_CTRL_GPU_OPTIMAL_CLOCK_FREQS - not supported
+#
+
+NV_CTRL_GPU_OPTIMAL_CLOCK_FREQS = 94  # not supported
+NV_CTRL_GPU_OPTIMAL_CLOCK_FREQS_INVALID = 0  # not supported
+
+#
+# NV_CTRL_GPU_OPTIMAL_CLOCK_FREQS_DETECTION - not supported
+#
+
+NV_CTRL_GPU_OPTIMAL_CLOCK_FREQS_DETECTION = 95  # not supported
+NV_CTRL_GPU_OPTIMAL_CLOCK_FREQS_DETECTION_START = 0  # not supported
+NV_CTRL_GPU_OPTIMAL_CLOCK_FREQS_DETECTION_CANCEL = 1  # not supported
+
+#
+# NV_CTRL_GPU_OPTIMAL_CLOCK_FREQS_DETECTION_STATE - not supported
+#
+
+NV_CTRL_GPU_OPTIMAL_CLOCK_FREQS_DETECTION_STATE = 96  # not supported
+NV_CTRL_GPU_OPTIMAL_CLOCK_FREQS_DETECTION_STATE_IDLE = 0  # not supported
+NV_CTRL_GPU_OPTIMAL_CLOCK_FREQS_DETECTION_STATE_BUSY = 1  # not supported
+
+#
+# NV_CTRL_FLATPANEL_CHIP_LOCATION - for the specified display device,
+# report whether the flat panel is driven by the on-chip controller,
+# or a separate controller chip elsewhere on the graphics board.
+# This attribute is only available for flat panels.
+#
+
+NV_CTRL_FLATPANEL_CHIP_LOCATION = 215  # R-DG
+NV_CTRL_FLATPANEL_CHIP_LOCATION_INTERNAL = 0
+NV_CTRL_FLATPANEL_CHIP_LOCATION_EXTERNAL = 1
+
+#
+# NV_CTRL_FLATPANEL_LINK - report the number of links for a DVI connection, or
+# the main link's active lane count for DisplayPort.
+# This attribute is only available for flat panels.
+#
+
+NV_CTRL_FLATPANEL_LINK = 216  # R-DG
+NV_CTRL_FLATPANEL_LINK_SINGLE = 0
+NV_CTRL_FLATPANEL_LINK_DUAL = 1
+NV_CTRL_FLATPANEL_LINK_QUAD = 3
+
+#
+# NV_CTRL_FLATPANEL_SIGNAL - for the specified display device, report
+# whether the flat panel is driven by an LVDS, TMDS, or DisplayPort signal.
+# This attribute is only available for flat panels.
+#
+
+NV_CTRL_FLATPANEL_SIGNAL = 217  # R-DG
+NV_CTRL_FLATPANEL_SIGNAL_LVDS = 0
+NV_CTRL_FLATPANEL_SIGNAL_TMDS = 1
+NV_CTRL_FLATPANEL_SIGNAL_DISPLAYPORT = 2
+
+#
+# NV_CTRL_USE_HOUSE_SYNC - when INPUT, the server (master) frame lock
+# device will propagate the incoming house sync signal as the outgoing
+# frame lock sync signal.  If the frame lock device cannot detect a
+# frame lock sync signal, it will default to using the internal timings
+# from the GPU connected to the primary connector.
+#
+# When set to OUTPUT, the server (master) frame lock device will
+# generate a house sync signal from its internal timing and output
+# this signal over the BNC connector on the frame lock device.  This
+# is only allowed on a Quadro Sync II device.  If an incoming house
+# sync signal is present on the BNC connector, this setting will
+# have no effect.
+#
+# This attribute may be queried through XNVCTRLQueryTargetAttribute()
+# using a NV_CTRL_TARGET_TYPE_FRAMELOCK or NV_CTRL_TARGET_TYPE_X_SCREEN
+# target.
+#
+
+NV_CTRL_USE_HOUSE_SYNC = 218  # RW-
```

Comment:
`.format()` is recommended way instead of string interpolation with `%`.

Reply:
Is this the right line for the comment? I see it pointing to `NV_CTRL_USE_HOUSE_SYNC = 218  # RW-` where there is no string interpolation.

Reply:
It looks like GitHub bug (the first one I've ever seen). :) I see correct line in the diff view: `return '<nVidia %s #%d>' % (self._name, self.id())` when I go to "Files changed" tab. But another line is displayed on main PR page for me as well.